library/src/main/java/pl/netigen/rewards/RewardDialogFragment.java

```diff
@@ -0,0 +1,335 @@
+package pl.netigen.rewards;
+
+import android.graphics.Color;
+import android.graphics.Point;
+import android.graphics.drawable.ColorDrawable;
+import android.os.Bundle;
+import android.view.Display;
+import android.view.Gravity;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.Window;
+import android.widget.FrameLayout;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatDialogFragment;
+import androidx.appcompat.widget.AppCompatTextView;
+import androidx.core.content.ContextCompat;
+
+import com.bumptech.glide.Glide;
+
+import java.util.List;
+
+import pl.netigen.netigenapi.BaseBannerActivity;
+import pl.netigen.netigenapi.R;
+
+public class RewardDialogFragment extends AppCompatDialogFragment {
+
+    private static final String TAG = "RewardDialogFragment";
+
+    public static double MATCH_PARENT = ViewGroup.LayoutParams.MATCH_PARENT;
```

Comment:
Unused:
public static double MATCH_PARENT
public static double WRAP_CONTENT

Reply:
It is used by library users to correctly set match_parent or wrap_content